v2/api/scheme.go

```diff
@@ -0,0 +1,17 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package api
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"github.com/Azure/azure-service-operator/v2/internal/controllers"
+)
+
+// CreateScheme is a helper method to export 'internal/controllers/controller_resources.go/CreateScheme()' method outside internal package for external use.
+func CreateScheme() *runtime.Scheme {
```

Comment:
Given the way Go imports are named (by default), usage of this becomes `api.CreateScheme()` - this reads very nicely. :+1: